src/javascripts/Services/referencedByViewService.js

```diff
@@ -4,10 +4,10 @@ ngapp.service('referencedByViewService', function($timeout, layoutService, setti
         scope.releaseTree = referencedByViewFactory.releaseTree;
 
         scope.buildTree = function() {
-            xelib.SetSortMode(scope.sort.column, scope.sort.reverse);
             scope.tree = xelib.GetReferencedBy(scope.record).map(function(handle) {
                 return scope.buildNode(handle);
             });
+            scope.sortTree();
```

Comment:
I dislike the fact that the referencedByViewService has functions with `tree` in their names.  We don't have a tree on this view, so it's kind of a misnomer.